website/docs/guides/core-to-cloud-1.md

```diff
@@ -0,0 +1,244 @@
+---
+title: Move from dbt Core to dbt Cloud - Getting started
```

Comment:
Not sure stylistically we should have dashes in the title. Maybe something like this (or other variations)
```suggestion
title: Get started with the move from dbt Core to dbt Cloud 
```

Reply:
not sure i'm a fan of this proposed one. we seem to have other characters (/ and -) on other guides. I think the existing version is clearer and matches the upcoming marketing site.